macOS-only failures in CI with RDKit visualization

Log: https://github.com/openforcefield/openff-toolkit/runs/5504391595?check_suite_focus=true

Traceback:
```
________________ TestMoleculeVisualization.test_visualize_rdkit ________________
openff/toolkit/tests/test_molecule.py:3797: in test_visualize_rdkit
    assert isinstance(mol.visualize(backend="rdkit"), IPython.core.display.SVG)
openff/toolkit/topology/molecule.py:6479: in visualize
    from rdkit.Chem.Draw import (  # type: ignore[import]
../../../miniconda3/envs/test/lib/python3.9/site-packages/rdkit/Chem/Draw/__init__.py:14: in <module>
    from rdkit.Chem.Draw import rdMolDraw2D
E   ImportError: dlopen(/Users/runner/miniconda3/envs/test/lib/python3.9/site-packages/rdkit/Chem/Draw/rdMolDraw2D.so, 2): Library not loaded: @rpath/libfontconfig.1.dylib
E     Referenced from: /Users/runner/miniconda3/envs/test/lib/libcairo.2.dylib
E     Reason: Incompatible library version: libcairo.2.dylib requires version 14.0.0 or later, but libfontconfig.1.dylib provides version 13.0.0
```

I can't reproduce this locally (macOS, Python 3.9) with the same versions of `cairo`, `pycairo`, and `rdkit`, the latter of which was recently bumped to 2021.09.5. I checked the RDKit and its feedstock's issue tracker and nobody else has reported it yet.

--- a/devtools/conda-envs/beta_rc_env.yaml
+++ b/devtools/conda-envs/beta_rc_env.yaml
@@ -16,8 +16,6 @@ dependencies:
   - networkx
   - ambertools
   - rdkit
-  # https://github.com/openforcefield/openff-toolkit/issues/1215#issuecomment-1078446482
-  - cairo=1.16.0=*_1010
   - xmltodict
     # Test-only/optional/dev
   - mdtraj

--- a/devtools/conda-envs/rdkit.yaml
+++ b/devtools/conda-envs/rdkit.yaml
@@ -18,8 +18,6 @@ dependencies:
   - networkx
   - ambertools
   - rdkit
-  # https://github.com/openforcefield/openff-toolkit/issues/1215#issuecomment-1078446482
-  - cairo=1.16.0=*_1010
   - packaging
     # Removed until a new release which targets openff-toolkit is made.
 #  - openmmforcefields

--- a/devtools/conda-envs/test_env.yaml
+++ b/devtools/conda-envs/test_env.yaml
@@ -17,8 +17,6 @@ dependencies:
   - networkx
   - ambertools
   - rdkit
-  # https://github.com/openforcefield/openff-toolkit/issues/1215#issuecomment-1078446482
-  - cairo=1.16.0=*_1010
   - xmltodict
   - python-constraint
     # Test-only/optional/dev

